emperor/support_files/js/util.js

```diff
@@ -83,6 +82,34 @@ function naturalSort(list){
   return result.concat(alphaPart, numericPart);
 }
 
+
+/**
```

Comment:
General comment about this, JavaScript uses camelcase instead of underscores, would you mind changing that for the variable names?